attributes/default.rb

```diff
@@ -134,8 +134,12 @@
 default['cluster']['slurm']['user_id'] = node['cluster']['reserved_base_uid'] + 1
 default['cluster']['slurm']['group'] = node['cluster']['slurm']['user']
 default['cluster']['slurm']['group_id'] = node['cluster']['slurm']['user_id']
-default['cluster']['slurm']['dbduser'] = 'slurmdbd'
-default['cluster']['slurm']['dbduser_id'] = node['cluster']['reserved_base_uid'] + 5
+default['cluster']['slurm']['dbduser'] = 'slurm'
+default['cluster']['slurm']['dbduser_id'] = if node['cluster']['slurm']['dbduser'].eql?(node['cluster']['slurm']['user'])
+                                              default['cluster']['slurm']['user_id']
+                                            else
+                                              node['cluster']['reserved_base_uid'] + 5
+                                            end
```

Comment:
@demartinofra what do you think, should we leave this in the hope that having two different users becomes functional in the future? Or do we just remove it from the code?

Reply:
As agreed we can just drop it for now